utils/checksec/Makefile

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (C) 2020 CZ.NIC z.s.p.o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=checksec.sh
+PKG_VERSION:=2.2.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/slimm609/checksec.sh/archive/$(PKG_VERSION)
+PKG_HASH:=d93c8ddd8bc1c3cbfbc28e34b3b3233f82f79d6800d55ae27dbea3d8b9933435
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE_FILES:=LICENSE.txt
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/checksec/default
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Utility to check PIE, RELRO, ASLR etc.
+  URL:=https://github.com/slimm609/checksec.sh
+  DEPENDS:=+bash
+endef
+
+define Package/checksec
+  $(call Package/checksec/default)
+  TITLE:=Utility to check PIE, RELRO, ASLR etc.
+  DEPENDS+=+file +binutils +procp-ps-sysctl +openssl-util +coreutils-stat
+endef
+
+define Package/checksec_automator
+  $(call Package/checksec/default)
+  TITLE:=Utility to use checksec for dirs
+  DEPENDS+=+checksec +coreutils-tee +findutils-find +grep
```

Comment:
Needs an additional dependency on +coreutils .

Reply:
I added coreutils dependency to checksec package